array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,22 @@
+/* exported drop */
+
+/*  function declaration with parameter array(an array)
+      and count (number of items in array to drop)
+    variable declaration for final array to return (is empty at first)
+    for loop where i is the indexes of parameter array to push items from parameter
+      array into final array starting from the index number equal to that of count parameter
+    if statement to account for empty parameter arrays, to return an empty array
+    return the final array from function
+*/
+
+function drop(array, count) {
+  var finalArr = [];
+  for (var i = count; i < array.length; i++) {
+    if (array.length === 0) {
+      finalArr = [];
```

Comment:
This conditional isn't necessary. If the array has a length of zero, then the loop would never start, which would mean you return the initial value of `finalArr`, which is an empty array.